WordPress/Classes/Networking/AccountServiceRemoteREST.m

```diff
@@ -60,8 +54,21 @@ - (void)getDetailsWithSuccess:(void (^)(NSDictionary *userDetails))success failu
           }];
 }
 
+
 #pragma mark - Private Methods
 
+- (RemoteUser *)remoteUserFromDictionary:(NSDictionary *)dictionary
+{
+    RemoteUser *remoteUser = [RemoteUser new];
+    remoteUser.userID = [dictionary numberForKey:@"ID"];
```

Comment:
We can use constants here - but since the keys are for the REST service only, I'd probably consider declaring those in this same file... even possibly in this very method if there's no expectation of reusing them.